ui/components-react/src/components-react/filter-builder/FilterBuilderToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from "react";
+import cx from "classnames";
+import { Flex, type FlexProps, IconButton } from "@itwin/itwinui-react";
+import { SvgAdd, SvgDelete } from "@itwin/itwinui-icons-react";
+import { UiComponents } from "../UiComponents";
+
+/**
+ * Props for [[PropertyFilterBuilderToolbar]] component.
+ * @internal
+ */
+interface PropertyFilterBuilderToolbarProps {
+  /** Size to render the component. If undefined, defaults to iTwinUI "medium" size. */
+  size?: "small" | "large";
+  /** Function to add child of current group. */
+  onAddChild?: () => void;
+  /** Function to remove child from current group. */
+  onDelete?: () => void;
+}
+
+/** Toolbar displaying the "add" and "delete" row buttons in the filter builder
+ * @beta
+ */
+export const PropertyFilterBuilderToolbar = (
+  props: FlexProps & PropertyFilterBuilderToolbarProps
```

Comment:
@GerardasB @raplemie Doesn't exposing `itwinui-react` types require to make `itwinui-react` as peer dependency?

Reply:
That's already a direct dependency, not sure why it would need to become a peer, not thrilled to expose these dependencies like that though, is this necessary ? iTwinUI appears to be hoping to move much faster that we actually want to ourselves with Major (`FlexProps` type is probably no longer exposed anyway in iTwinUI 3.0 as they removed most of their props types...)

I'm not sure what all the FlexProps are, but I doubt that it make much sense at this point to let user change these for the toolbar, these should be uniformized, not free for all I think